src/app/models/HarvestItem.ts

```diff
@@ -35,17 +35,46 @@ export type HarvestItemState =
   | "completed"
   | "errored";
 
+export type ValidationFixable =
+  | "missingDate"
+  | "ambiguousDateTime"
+  | "futureDate"
+  | "missingUploader"
+  | "noSiteId";
+
+export type ValidationNotFixable =
+  | "doesNotExist"
+  | "notAFile"
+  | "fileEmpty"
+  | "invalidExtension"
+  | "noDuration"
+  | "tooShort"
+  | "channelCount"
+  | "sampleRate"
+  | "bitRate"
+  | "mediaType"
+  | "duplicateFile"
+  | "duplicateFileInHarvest"
+  | "notAllowedToUpload"
+  | "overlappingFilesInHarvest"
+  | "overlappingFiles";
```

Comment:
I haven't seen where this is used yet, but me should make sure this doesn't fail if the server produces new validations.

I don't think it will since the validations are produced by the server and typescript does not check these at runtime.

Reply:
We use this to type check the conditions in the validation widget. Changing these names will not break anything, however, keeping them synced up would be nice